### ISSUE TITLE
Malware detection

### ISSUE THREAD

**Issue by FndNur1Labs** [MintcoinCommunity/Mintcoin-Desktop-Wallet#100]
OK, issue one, some of the confusion is that v2.0.2 (from 2016?) on page one of this thread is old and scans as a threat, not necessarily a valid threat, possibly bitcoin miner type hits.  A version that should NOT be linked in the opening post on the official thread for Mintcoin.  Regardless, the 2.0.2 is not the issue at hand.

Next issue, two, the links posted going back the last 5 or so posts for (MintCoin-2.1.0-win32.zip) has a MintCoin-Qt.exe that was updated on github on October 28, 2018 with an incorrect size file of 8,428,482 which is likely a virus.  This repository should be taken down immediately (if possible).
Many thanks to desynct for catching this.

Housekeeping note; I'm editing all links I posted recently to the corrupted file and "killing" the link for security purposes, but otherwise leaving it intact to cut down on future confusion. 

[Check here](https://bitcointalk.org/index.php?topic=450381.new#new)

**Comment by shane-kerr** [comment]
I've removed the Windows installer and ZIP file from the release until we can investigate.

It does look like it was changed since the release.

I've removed all users other than myself and @EuroCentiem-dev from the MintCoin Community group, and reset my own password.

I have contacted GitHub to see if they can give us a log of which account made the changes and when.

I'll get more details when I am able to, which will be tonight.
